### DISCUSSION TITLE
Dreame S10 Pro Plus 热水版 doesn't allow me to go past this step even after connecting to my Dreamehome account

### DISCUSSION THREAD

**Discussion by changteck** [Tasshack/dreame-vacuum#195] [Q&A] [answered]
![image](https://github.com/Tasshack/dreame-vacuum/assets/73820284/0145a3d2-bed6-4e68-9c1f-775dcfd6dcad)

This particular model isn't supported by Xiaomi Home, so I tried the beta version which has Dreame Cloud.
Unfortunately, it says no devices are supported by the account even though my account has the device added.

What can I do?

**Comment by Tasshack** [comment]
Yes it is supported by this integration too but i think there is a bug preventing it to be added to the HA.

**Reply by changteck** [reply]
Thank you, not to rush you, but any estimated timeline on fixing the bug?
Happy to wait!

**Reply by Tasshack** [reply]
It would be very fast if you are willing to share your device to my dreamehome account.

**Reply by changteck** [reply]
Yes of course! Let me know details. Man I'm willing to donate some money to you as well as thanks!

**Reply by Tasshack** [reply]
My dreamehome account user id is `BP428398`. You can share your device from Me -> Device Sharing menu without giving the camera access.

**Reply by changteck** [reply]
Shared bro, thanks! Do you have a crypto wallet? I want send you some ETH/USDT!

**Reply by Tasshack** [reply]
Ok I found the problem, the device was not in the supported devices list simply because it is too new. I have added the device to the supported devices list with its map private key and it is working with the integration now.
I cannot control the device from the integration right now because it is region locked. You need to bypass the region lock using VPN/DNS method first to make the map work with the integration too.

I don't use crypto but a paypal donation is much appreciated.

<a href="https://paypal.me/tasshackK" target="_blank"><img src="https://www.paypalobjects.com/webstatic/mktg/logo/pp_cc_mark_37x23.jpg" border="0" alt="PayPal Logo" style="height: auto !important;width: auto !important;"></a>

You can remove the device access from my account, I don't need it anymore.

**Reply by changteck** [reply]
Thanks bro! You are a life saver! I will try it now, and once it is successful, will make a donation!

Also, can you advise how do I bypass the region lock using DNS method?
I have tried the VPN method but it doesn't work.

Any guide would be appreciated!

**Reply by changteck** [reply]
@Tasshack I have donated some money to you!
Thanks again! I haven't tried it, but appreciate it.

Also, waiting for you to let me know how to bypass region lock using the DNS/VPN method.

**Reply by Tasshack** [reply]
I don't own a device with the region lock so I don't know actually how to do it but the app should work when you do it correctly.
Currently, region unlocking methods are kept like secret to prevent Dreame knowing and blocking them but I am sure you will find a solution that works if you search deep enough trough the internet.

**Reply by changteck** [reply]
Thanks so much.

Also, I have tried the integration again - it still says I have no devices that are supported in the region selected (CN).

Do I have to reinstall HA? Or do I have to wget your repo again?

**Reply by Tasshack** [reply]
You need to run the manual install command again to update the integration then it should work.

**Comment by Tasshack** [comment, marked as answer]
S10 Pro Plus (Hot Water) - dreame.vacuum.r2360 is added to the supported devices list on latest test release.
https://github.com/Tasshack/dreame-vacuum/releases/tag/v2.0.0b2
